README.md

```diff
@@ -43,7 +43,7 @@ Required dependencies ROS Kinetic, navigation, OpenVDB, TBB.
 
 ### OpenVDB
 
-`sudo apt-get install libopenvdb3.1 libopenvdb-doc libopenvdb-dev libopenvdb-tools`
+`sudo apt-get install libopenvdb3.1 libopenvdb-doc libopenvdb-dev libopenvdb-tools`libopenexr22 libopenexr-dev
```

Comment:
please place in the openexr section, replacing what I put, this is correct, I messed up